packages/dd-trace/src/telemetry/logs/log-collector.js

```diff
@@ -27,7 +27,7 @@ function createHash (logEntry) {
 }
 
 function isValid (logEntry) {
-  return logEntry?.level && logEntry.message
+  return logEntry?.level && logEntry.message && logEntry.transmit !== false
```

Comment:
Maybe I'm reading the code wrong, but isn't `transmit` only a property of the `LogConfig` instance? A `Log` instance would have `sendViaTelemetry` set instead, right?

```suggestion
  return logEntry?.level && logEntry.message && logEntry.sendViaTelemetry !== false
```